components/tikv_util/src/sys/cpu_time.rs

```diff
@@ -122,16 +175,172 @@ mod imp {
             Ok(ret)
         }
     }
+
+    pub fn cpu_time() -> io::Result<std::time::Duration> {
+        let mut time = unsafe { std::mem::zeroed() };
+
+        if unsafe { libc::getrusage(libc::RUSAGE_SELF, &mut time) } == 0 {
```

Comment:
`getrusage` will return the resource usage since the process started? Is this expected?